Removed NodePulseEffect.

rnetiks opened the pull request:
Replaced DOTween with custom scaling logic for node animations, simplifying behavior and enabling runtime toggling. Adjusted configuration change handling to reset effects dynamically, improving flexibility and user experience.

rnetiks committed:
Refactor NodePulseEffect and improve animation handling.

Replaced DOTween with custom scaling logic for node animations, simplifying behavior and enabling runtime toggling. Adjusted configuration change handling to reset effects dynamically, improving flexibility and user experience.

krypto5863 commented:
Can't say I like reinventing the wheel. But node pulsing wasn't great idea. And with the release of object map, I saw a performance hit when too many nodes are around. I don't know if your implementation resolves that. If it doesn't, it's better to just remove the feature entirely, it's not that helpful to excuse it's existence.

Also these whitespaces are messed up now, make sure you're using tabs.

krypto5863 reviewed (changes requested):
As mentioned above.

rnetiks committed:
Remove NodePulseEffect feature and related animations

The NodePulseEffect class and its functionality, including configuration and references, have been removed from the project. This simplifies the codebase by focusing on other node-related features like tooltips, improving maintainability.

krypto5863 reviewed (approved):
Looks good.
